### DISCUSSION TITLE
In-game cursor feels laggy/choppy when holding gameplay buttons

### DISCUSSION THREAD

**Discussion by vnvz** [ppy/osu#25281] [Q&A]
### Type

Performance

### Bug description

During gameplay, when holding any of the gameplay buttons (B1 or B2) the cursor starts to lag. This behavior occurs independently of raw input being on or off.

System specs:
- Windows 11 Pro 22H2 (22621.2428)
- Ryzen 5 5600G
- Radeon RX570 (AMD Drivers 23.9.2)
- 16GB RAM (2x8GB) 2400MHz

### Screenshots or videos

https://github.com/ppy/osu/assets/58855912/6df6e0d0-51cf-49b9-b461-237f26e9721b

### Version

2023.1026.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/13193217/database.log)
[input.log](https://github.com/ppy/osu/files/13193218/input.log)
[network.log](https://github.com/ppy/osu/files/13193219/network.log)
[performance.log](https://github.com/ppy/osu/files/13193220/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13193221/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13193222/updater.log)

**Comment by peppy** [comment]
Could you please take the same video with the performance graphs visible? Ctrl+F11 twice to show them.

**Reply by vnvz** [reply]
It seems that after rebooting my computer since then, the problem disappeared. 
https://github.com/ppy/osu/assets/58855912/7f8d9332-eb14-423d-861b-e562d943a4a8

I'm also sending the logs from the same session I recorded this new clip.
[input.log](https://github.com/ppy/osu/files/13196311/input.log)
[network.log](https://github.com/ppy/osu/files/13196312/network.log)
[performance.log](https://github.com/ppy/osu/files/13196313/performance.log)
[runtime.log](https://github.com/ppy/osu/files/13196314/runtime.log)
[updater.log](https://github.com/ppy/osu/files/13196315/updater.log)
[database.log](https://github.com/ppy/osu/files/13196310/database.log)

I don't know if I might me able to reproduce the same behavior again, but at the moment it seemed right thing to do to report this.
